sdk/face/ai-vision-face-rest/README.md

```diff
@@ -18,24 +18,20 @@ The Azure AI Face service provides AI algorithms that detect, recognize, and ana
 
 ## Getting started
 
-### Currently supported environments
-
-- LTS versions of Node.js
-
-See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
-
 ### Prerequisites
 
-- You need an [Azure subscription](https://azure.microsoft.com/free/) to use this package and either
-  - an [Azure Face account](https://portal.azure.com/#blade/Microsoft_Azure_ProjectOxford/CognitiveServicesHub/Face) or
-  - an [Azure Cognitive Service account](https://portal.azure.com/#view/Microsoft_Azure_ProjectOxford/CognitiveServicesHub/~/AllInOne)
+- Node.js 18 or above.
```

Comment:
Any reason to change the LTS to Node 18? I mean Node 18 may be deprecated someday.

Reply:
Just considered a case that in the future that v18 is deprecated, a service has been running well for a long time using v18 and wants to introduce a new face feature using this SDK. But this might be too rare for us to address.